src/components/organisms/UiList/_internal/UiListItem.vue

```diff
@@ -2,6 +2,8 @@
   <li
     v-bind="listItemAttrs"
     class="ui-list-item"
+    role="option"
```

Comment:
@sonya0504 @kubaszajna should we add it by default? Why should the basic list provide `role` and `aria-selected`?